Makefile

```diff
@@ -1,21 +1,32 @@
-SKIP_SQUASH?=0
-VERSIONS="2.4.41"
+OPENLDAP_VERSION := $(shell git describe --tags --always --dirty)
+DOCKER_REGISTRY := dockerhub.com
```

Comment:
shouldn't we use quay instead? Do we have the images present?

Also `s/DOCKER/CONTAINER/` please

Reply:
I could change that one to `quay.io`, but the containers were uploaded to dockerhub.com.